src/gui/include_MINI/gui_config_printer.hpp

```diff
@@ -1,6 +1,12 @@
-// gui_config_mini.h
+/**
+ * @file gui_config_printer.hpp
+ * @brief printer specific gui config
+ */
```

Comment:
Is this new part? I think we've agreed not to have these headers / not have the `@file` part (because it doesn't particularly do anything in doxygen anyway).

Goes for other files too...

Reply:
I don't know anything about agreement not having this headers.